### PR TITLE
Restore monospace font in commit and merge title fields

### LINKED ISSUE #5188: Restore `monospace-textareas`/`suggest-commit-title-limit`
### Description

Part feature request, part bug.

One of our features was to use fixed-width fonts in the commit fields to know when it reaches the end: `suggest-commit-title-limit`

This no longer works. We should probably restore `monospace-textareas` in some capacity.

The last relevant piece of code still lives here:

https://github.com/refined-github/refined-github/blob/f843b8b97605a9f0b7d0b810f2ac1aad6bace373/source/features/github-bugs.css#L45-L47

And one selector was removed in in: [`5b7282a` (#5113)](https://github.com/refined-github/refined-github/pull/5113/commits/5b7282afc40b013f5928271fb6740cf70b4e4295)

### Screenshot

Screen Shot

### Example URL

https://github.com/refined-github/refined-github/pull/5139

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes #5188 

## Test URLs

 * [Commit title field](https://togithub.com/refined-github/refined-github/edit/main/readme.md)
 * [Open PR merge title field](https://togithub.com/refined-github/sandbox/pull/4)

## Screenshot

image

### CODE REVIEW

File: source/features/github-bugs.css [hunk 1]
```diff
@@ -42,6 +42,8 @@
 	font-weight: 600;
 }
 
+#merge_title_field,
+#commit-summary-input, /* Restore monospace font in commit/merge title fields #5188 */
 :is(.new_public_key, .new_gpg_key) textarea { /* Monospace textareas for new SSH/GPG keys #4917 */
 	font-family: ui-monospace, SFMono-Regular, 'SF Mono', Menlo, Consolas, 'Liberation Mono', monospace !important;
 }
```
Comment (cheap-glitch) [inline source/features/github-bugs.css:49]: Should this be tied to `suggest-commit-title-limit` instead?

Comment (fregante) [reply source/features/github-bugs.css:49]: Yes but I think it's fine as is.

**Comment by fregante** [comment]
LGTM, but before merging:

- can you ensure that the field size still matches the expected title length?
- hotfix.css too plz 😁

**Comment by cheap-glitch** [comment]
> can you ensure that the field size still matches the expected title length?

https://user-images.githubusercontent.com/46634000/146538933-93acc198-dc97-45be-860a-a6afef4b567a.mp4

**Pull request merged**